integration/log_rotation_test.go

```diff
@@ -111,6 +120,13 @@ func (s *LogRotationSuite) TestTraefikLogRotation(c *check.C) {
 	c.Assert(lineCount, checker.GreaterOrEqualThan, 7)
 }
 
+func logAccessLogFile(c *check.C, fileName string) {
+	output, err := ioutil.ReadFile(fileName)
+	c.Assert(err, checker.IsNil, check.Commentf("error opening access log file %s: %s", fileName, err))
+	c.Logf("Contents of file %s", fileName)
+	c.Log(string(output))
```

Comment:
Could also be part of the previous statement, separating the message "header" and the content by a newline (`\n`).

Reply:
Simplified it into one line 👍